City-of-Helsinki/helsinki-paatokset
# PP-381: Change official label to use tag styles.

**To test**
- Checkout branch, run `make drush-cr drush-cim`
- Import decisionmaker data if you don't have any locally: `drush migrate-import ahjo_decisionmakers:all --update`
- Check some officials (viranhaltijat) on the decisionmaker page: https://helsinki-paatokset.docker.so/fi/paattajat
  - The sector and top level organization name should appear with a tag style (similar to tags on decision pages)
  - Edit an official page and remove one or both values. Make sure an empty tag isn't printed on the page
